cmd/cli/mesh_list.go

```diff
@@ -62,17 +63,47 @@ func (l *meshListCmd) run() error {
 	}
 
 	w := newTabWriter(l.out)
-	fmt.Fprintln(w, "MESH NAME\tNAMESPACE\t")
+	nds, _ := l.clientSet.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
+
+	fmt.Fprintf(w, "OSM Deployments on cluster %s\n", nds.Items[0].GetName())
```

Comment:
This line might be confusing since it displays the name of one node instead of the Kubernetes cluster the OSM controller is running on. I'd suggest we either figure out how to display the name of the cluster (via Kubernetes context info) or remove this line for now and include the cluster name info in a separate PR.

Reply:
hi @michelleN so should I remove the cluster name line if it can be included in separate PR?

Reply:
yes that'd be great